.circleci/config.yml

```diff
@@ -25,6 +25,9 @@ aliases:
         command: |
           /bin/bash -c 'echo -e "[mysqld]\ndatadir = /dev/shm" > /etc/my.cnf.d/obs.cnf && cp -a /var/lib/mysql/* /dev/shm && /usr/lib/mysql/mysql-systemd-helper start'
         name: db
+      - image: registry.opensuse.org/obs/server/unstable/container/leap/42.3/images/openbuildservice/backend:latest
+        command: /obs/contrib/start_development_backend -d /obs
+        name: backend
```

Comment:
@ChrisBr I need your input here, this does not work

Reply:
The problem is that only the "main" container does have a git checkout, so the start_development_backend file does not exist at the time you want to start the container :cry: 

Hm, I would need to investigate how we could solve that...

One obvious solution would be to build the container with the backend inside :thinking:

Reply:
For minitest, we run the backend in the same container as the frontend (for other reasons because we need access to the filesystem of the backend in the minitests). This could be a **workaround** for testing.

Other solution could be that we use docker-compose in CircleCI, similar to what we do in Travis. This also would bring the benefit of unifying the two different CI systems....

Reply:
I'd love that actually - the circleci config file is so much different than the other :(